### PR TITLE
Add a script to automate the setup of a local shelly cluster

### PR DESCRIPTION
**Pull request by dcoutts**
It makes all the static files needed to run a couple BFT nodes and a
stake pool node, and to set up the stake pool registration and
delegation to delegate all stake to the stake pool.

### CODE REVIEW

File: scripts/shelley-from-scratch/mkfiles.sh [hunk 1]
```diff
@@ -0,0 +1,338 @@
+#!/bin/bash
```
Comment (disassembler) [inline scripts/shelley-from-scratch/mkfiles.sh:1]:
```suggestion
#!/usr/bin/env bash
```

Comment (disassembler) [reply scripts/shelley-from-scratch/mkfiles.sh:1]: this lets the script work when bash isn't in `/bin`

File: scripts/shelley-from-scratch/mkfiles.sh [hunk 2]
```diff
@@ -0,0 +1,338 @@
...
+
+# Then edit the genesis.spec.json ...
+
+SUPPLY=1000000000
+
+# We're going to use really quick epochs (300 seconds), by using short slots 0.2s
+# and K=10, but we'll keep long KES periods so we don't have to bother
+# cycling KES keys
+sed -i ${ROOT}/genesis.spec.json \
+    -e 's/"slotLength": 1/"slotLength": 0.2/' \
```
Comment (disassembler) [inline scripts/shelley-from-scratch/mkfiles.sh:38]:
```suggestion
    -e 's/"slotLength": 1/"slotLength": 0.1/' \
```

Comment (disassembler) [reply scripts/shelley-from-scratch/mkfiles.sh:38]: I've been able to run this at 0.1 and then it can be tested twice as fast.

Comment (dcoutts) [reply scripts/shelley-from-scratch/mkfiles.sh:38]: It makes me nervous. 3min per epoch is pretty quick already :-)

**Comment by dcoutts** [comment]
bors merge

**Pull request merged**